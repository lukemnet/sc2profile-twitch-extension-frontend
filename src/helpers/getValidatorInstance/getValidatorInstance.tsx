import * as Yup from 'yup';

const getValidator = () => Yup;

export default getValidator;
