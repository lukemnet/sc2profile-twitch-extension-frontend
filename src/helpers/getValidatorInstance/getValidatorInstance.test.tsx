import getValidatorInstance from './getValidatorInstance';

it('returns Yup-compatible validator instance', () => {
  expect(getValidatorInstance()).toMatchSnapshot();
});