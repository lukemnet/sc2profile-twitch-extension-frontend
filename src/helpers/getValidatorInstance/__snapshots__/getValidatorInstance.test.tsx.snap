// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns Yup-compatible validator instance 1`] = `
Object {
  "ValidationError": [Function],
  "__esModule": true,
  "addMethod": [Function],
  "array": [Function],
  "bool": [Function],
  "boolean": [Function],
  "date": [Function],
  "isSchema": [Function],
  "lazy": [Function],
  "mixed": [Function],
  "number": [Function],
  "object": [Function],
  "reach": [Function],
  "ref": [Function],
  "setLocale": [Function],
  "string": [Function],
}
`;
